### PR TITLE
Disable Style/NumericLiterals by default

### PR DESCRIPTION
**Pull request by mssola**
I've also added the rationale for this in the Ruby.md file

### CODE REVIEW

File: Ruby.md [hunk 1]
```diff
@@ -585,3 +585,9 @@ quotes for better consistency.
 ### Style/StringLiteralsInInterpolation
 
 In interpolation double quotes are more practible, so prefer using them.
+
+### Style/NumericLiterals
+
+Rubocop's default value is to always put underscores on large numeric literals.
```
Comment (aspiers) [inline Ruby.md]: Do you mean Rubocop's default behaviour is to complain about the lack of underscores on large numberic literals?  Presumably it only analyzes code and doesn't actually change anything ;-)  So I'd suggest slightly rewording this.

Comment (mssola) [inline Ruby.md]: @aspiers fixed, thanks.

**Comment by aspiers** [comment]
+1

**Comment by rsalevsky** [comment]
+1

**Comment by mssola** [comment]
@cornelius ping

**Pull request merged**